src/domains/cdc/_modules/apim/named_values_cdc_support.tf

```diff
@@ -0,0 +1,25 @@
+resource "azurerm_api_management_named_value" "io_fn_cdc_support_url" {
+  name                = "io-cdc-support-func-url"
+  api_management_name = data.azurerm_api_management.apim.name
+  resource_group_name = data.azurerm_api_management.apim.resource_group_name
+  display_name        = "io-cdc-support-func-url"
+  value               = "https://io-p-itn-cdc-support-func-01.azurewebsites.net"
```

Comment:
Could you read this value from a data o resource instead of hardcoding it?

Reply:
it would be almost the same value to get a data.